src/renderer/components/editorWithTabs/sourceCode.vue

```diff
@@ -129,6 +130,10 @@
     margin: 50px auto;
     max-width: 860px;
   }
+  .dark .source-code .CodeMirror {
```

Comment:
I think 

```css
.dark.source-code {
    background: var(--darkBgColor);
}
```

is OK, and you can delete 

```css
.dark {
    background: var(--darkBgColor);
}
```
now

Reply:
`.dark .source-code .CodeMirror` is needed too because it overwrite the background from CodeMirror.

Reply:
Do you have a space between `.dark` and `.source-code`?

Reply:
Yes, `editor-with-tabs` has the theme name and `source-code` not.

Reply:
But you have add a `:class="theme"` to `sourceCode.vue`? So I think `.dark.source-code` is OK?

Reply:
I run you PR locally, and  found that:

I think you have added theme's classname inside the `sourceCode.vue` file. So I don't think it is necessary to reference the external classname of the theme(.dark) in `editorWithTab/index.vue`.

Yes, `.CodeMirror` is necessary.

So I think css below is better?

```css
  .dark.source-code,
  .dark.source-code .CodeMirror  {
    /* Overwrite CodeMirror background color */
    background: var(--darkBgColor);
  }
/*.dark {
    background: var(--darkBgColor);
}*/
```

and delete `:class="theme"` in `editorWithTab/index.vue`.